data/blog/tevents.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: 'tevents: event logger and job monitor for tailnets'
+date: '2023-01-28'
```

Comment:
Should be 2023-02-28?

Reply:
Updated in 2b5ac0a69b338f35fc0c6eed450ff7974cefa2df